framework/source/class/qx/test/Mixin.js

```diff
@@ -240,8 +240,8 @@ qx.Class.define("qx.test.Mixin",
           {
             var s = "";
             if (!dontRecurs) {
-              this.__b = new qx.Patch2();
-              s += "++" + this.__b.foo(true) + "__";
+              this['__b'] = new qx.Patch2();
```

Comment:
@derrell The fact is if we declare it as a member, then the exception https://github.com/qooxdoo/qooxdoo/blob/master/framework/source/class/qx/test/Mixin.js#L258 is thrown. But why? The class qx.Patch1 where qx.MPatch is included, does not have a private member __b itself. And, as discussed with @oetiker private members are valid in mixins.

What if in the case of the test, the intention was not to write clean code, but trick the testsuite somehow to be able to implement that test? In this case it may be valid to have a private variable not be declared member and hide it from the linter via a @lint hint.

We could merger @thron7 's PR https://github.com/qooxdoo/qooxdoo/pull/9164 first, leave the code as it was, asking @pcdummy t revert this changes and to rebase this PR after merging https://github.com/qooxdoo/qooxdoo/pull/9164, which also includes the fix by adding the lint hint.